libs/java/server_common/src/main/java/com/yahoo/athenz/common/server/util/config/ConfigManager.java

```diff
@@ -0,0 +1,463 @@
+/*
+ * Copyright 2017 Yahoo Inc.
```

Comment:
Fix copyright in all new files to "Copyright The Athenz Authors"

Reply:
Fixed